### ISSUE TITLE
Be more flexible about location of SSL certs on Linux

### ISSUE THREAD

**Issue by Laeeth** [symmetryinvestments/imap-d#70]
On Ubuntu 20 my certs is at `/etc/ssl/certs/ca-certificates.crt` and needs to be copied to `/etc/ssl/cert.pem` for the plugin to work.

**Comment by otrho** [comment]
On my Arch under `/etc/ssl` I have:
```
lrwxrwxrwx 1 root root    46 Jul 29 02:36 cert.pem -> ../ca-certificates/extracted/tls-ca-bundle.pem
```
So it installs a symlink itself by the looks of it.

This ServerFault answer is useful.  Maybe we can mirror what Go does and look everywhere until we find them.  https://serverfault.com/a/722646

**Comment by Laeeth** [comment]
Above makes sense.

**Comment by Laeeth** [comment]
closed by #77
